tests/integrations/test_dsm.py

```diff
@@ -182,15 +190,16 @@ class Test_DsmSQS:
     """ Verify DSM stats points for AWS Sqs Service """
 
     def setup_dsm_sqs(self):
-        self.r = weblog.get("/dsm?integration=sqs")
+        self.r = weblog.get("/dsm?integration=sqs&timeout=60", timeout=61)
 
-    @bug(weblog_variant="flask-poc", reason="DSM checkpoints for AWS SQS from dd-trace-py are not being receieved.")
+    @bug(weblog_variant="flask-poc", reason="DSM checkpoints for AWS SQS from dd-trace-py are not being received.")
+    @bug(weblog_variant="express4", reason="DSM checkpoints for AWS SQS from dd-trace-js are not being received.")
```

Comment:
is there a jira issue related with this bug?

Reply:
I'm not sure, I will be looking into this issue after and will either fix it with another PR or create an issue if I can't find the cause

Reply:
ok